### PR TITLE
fetch data from scene.json

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: _posts/development-guide/2018-02-26-scene-metadata.md [hunk 1]
```diff
@@ -212,3 +212,48 @@ As clear use cases start to emerge, we plan to define more conventions over the
...
+  log('parcels: ', parcel.land.sceneJsonData.scene.parcels)
+  log('base parcel: ', parcel.land.sceneJsonData.scene.base)
+
+  // spawn points
+  log('spawnpoints: ', parcel.land.sceneJsonData.spawnPoints)
+
+  // general scene data
+  log('title: ', parcel.land.sceneJsonData.display.title)
+  log('author: ', parcel.land.sceneJsonData.contact.name)
+  log('email: ', parcel.land.sceneJsonData.scene.contact.email)
```
Comment (nchamo) [inline _posts/development-guide/2018-02-26-scene-metadata.md]:
```suggestion
  log('email: ', parcel.land.sceneJsonData.contact.email)
```

File: _posts/development-guide/2018-02-26-scene-metadata.md [hunk 2]
```diff
@@ -212,3 +212,48 @@ As clear use cases start to emerge, we plan to define more conventions over the
...
+
+  // other info
+  log('all policy fields: ', parcel.land.sceneJsonData.policy)
+  log('rating: ', parcel.land.scene.sceneJsonData.contentRating)
+  log('blacklist: ', parcel.land.scene.sceneJsonData.blacklist)
```
Comment (nchamo) [inline _posts/development-guide/2018-02-26-scene-metadata.md]: We didn't include these on the type definition
```suggestion
```

**Review by nchamo — APPROVED** [review]
Looks good, just suggested a few corrections

File: _posts/development-guide/2018-02-26-scene-metadata.md (hunk 1 above)
Comment (nearnshaw) [reply _posts/development-guide/2018-02-26-scene-metadata.md]: good catch!

File: _posts/development-guide/2018-02-26-scene-metadata.md (hunk 2 above)
Comment (nearnshaw) [reply _posts/development-guide/2018-02-26-scene-metadata.md]: ok, good point
yeah, let's leave them out, I don't think they're important today anyway
they might have more use eventually, but not for now, so let's not complicate things unnecessarily

**Pull request merged**